app/uk/gov/hmrc/integrationcatalogueautopublish/models/exception/AutopublishException.scala

```diff
@@ -17,3 +17,12 @@
 package uk.gov.hmrc.integrationcatalogueautopublish.models.exception
 
 abstract class AutopublishException(message: String, cause: Throwable) extends RuntimeException(message, cause)
+
+object AutopublishException {
+
+  def addContext(message: String, context: Seq[(String, AnyRef)]): String = {
+    val contextMessage = context.map(value => s"${value._1}=${value._2}").mkString("; ")
```

Comment:
not sure if this is preferred 
```suggestion
    val contextMessage = context.map { case (key, value) => s"$key=$value" }.mkString("; ")
```